apps/prairielearn/src/pages/studentInstanceQuestion/studentInstanceQuestion.ts

```diff
@@ -171,7 +173,7 @@ async function validateAndProcessSubmission(req: Request, res: Response) {
 
 router.post(
   '/',
-  asyncHandler(async (req, res) => {
+  typedAsyncHandler<'instance-question', { client_fingerprint_id: string }>(async (req, res) => {
```

Comment:
This second parameter specifies extras, e.g. this goes through a fingerprint middleware.